plugins/tours/tests/phpunit/unit/_renderCustomColumnContent.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Tests for _render_custom_column_content().
+ *
+ * @package     spiralWebDb\CornerstoneTours\Tests\Unit
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Unit;
+
+use Mockery as m;
+use Brain\Monkey;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+use function spiralWebDb\CornerstoneTours\_render_custom_column_content;
+
+/**
+ * Class Tests_RenderCustomColumnContent
+ *
+ * @package spiralWebDb\CornerstoneTours\Tests\Unit
+ * @group   tours
+ * @group   admin
+ */
+class Tests_RenderCustomColumnContent extends Test_Case {
+
+	/**
+	 * Instance of the post object for each test.
+	 *
+	 * @var Mockery
+	 */
+	protected $post;
+
+	/**
+	 * The post ID
+	 *
+	 * @var integer
+	 */
+	protected $ID;
+
+	/**
+	 * The post type
+	 *
+	 * @var string
+	 */
+	protected $post_type;
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once TOURS_ROOT_DIR . '/src/admin/wp-list-table.php';
```

Comment:
Move the loading of this file to `setUpBeforeClass` like this:

```php
public static function setUpBeforeClass() {
	parent::setUpBeforeClass();

	require_once TOURS_ROOT_DIR . '/src/admin/wp-list-table.php';
}
```

Reply:
@hellofromtonya Updated.